tests/integration/bond_test.py

```diff
@@ -369,3 +370,17 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+def test_create_vlan_over_a_bond_slave(bond99_with_slave):
+    vlan_bond_slave_iface = bond99_with_slave[Interface.KEY][0][
```

Comment:
The name is confusing, how about: `bond_slave_ifname`.

Also, please make that long broken line readable by using intermediate variables:
```
bond_ifstate = bond99_with_slave[Interface.KEY][0]
bond_slave_ifname = bond_ifstate[Bond.CONFIG_SUBTREE][Bond.SLAVES][0]
```